src/app/heap_usage/values.ml

```diff
@@ -41,6 +41,26 @@ let zkapp_command =
   in
   List.hd_exn zkapp_commands
 
+let zkapp_proof =
+  List.fold_until
+    (Mina_base.Zkapp_command.all_account_updates_list zkapp_command)
+    ~init:None
+    ~f:(fun _acc a ->
+      match a.Mina_base.Account_update.authorization with
+      | Proof proof ->
+          Stop (Some proof)
+      | _ ->
+          Continue None )
+    ~finish:Fn.id
+  |> Option.value_exn
+
+let verification_key =
```

Comment:
could you just use the dummy vk, won't it be the same size?

Reply:
They were showing different sizes- Added dummy proof and vk as well to the app